guacamole/src/main/frontend/src/app/settings/controllers/connectionHistoryPlayerController.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * The controller for the general settings page.
+ */
+angular.module('manage').controller('connectionHistoryPlayerController', ['$scope', '$injector', 
+        function connectionHistoryPlayerController($scope, $injector) {
+
+    // Required services
+    var authenticationService = $injector.get('authenticationService');
+    var $routeParams          = $injector.get('$routeParams');
+
+    /**
+     * The identifier of the currently-selected data source.
+     *
+     * @type String
+     */
+    var dataSource = $routeParams.dataSource;
+
+    var recordingIdentifier = $routeParams.identifier;
```

Comment:
Why does `dataSource` get a jsdoc but none of these other params do?

Reply:
They definitely should be doc'd. I believe I just fixed this now via rebase (removed the variables entirely).